packages/redux/src/bags/actionTypes.ts

```diff
@@ -67,3 +67,25 @@ export const UPDATE_BAG_ITEM_REQUEST =
  */
 export const UPDATE_BAG_ITEM_SUCCESS =
   '@farfetch/blackout-redux/UPDATE_BAG_ITEM_SUCCESS';
+
+/**
+ * Action type dispatched when the fetch bag operation request fails.
+ */
+export const FETCH_BAG_OPERATION_FAILURE =
+  '@farfetch/blackout-core/FETCH_BAG_OPERATION_FAILURE';
```

Comment:
```suggestion
  '@farfetch/blackout-redux/FETCH_BAG_OPERATION_FAILURE';
```

This applies to the remaining action types.

Reply:
Done!